tests/common.cpp

```diff
@@ -0,0 +1,229 @@
+#include "common.h"
+#include <cmath>
+#include <cstdint>
+
+namespace SSE2NEON
+{
+int32_t nan = ~0;
```

Comment:
How about renaming `nan` to `NaN`? Thus, it can avoid potential symbol conflicts with [nan(3)](https://man7.org/linux/man-pages/man3/nan.3.html).